test/unit/presenters/publishing_api_presenters/classification_test.rb

```diff
@@ -0,0 +1,34 @@
+require 'test_helper'
+
+class PublishingApiPresenters::ClassificationTest < ActiveSupport::TestCase
+  test "presents a valid placeholder 'topic' content item" do
+    topic = create(:topic, name: "Defence and armed forces")
+    public_path = "/government/topics/defence-and-armed-forces"
+
+    expected = {
+      base_path: public_path,
+      publishing_app: "whitehall",
+      rendering_app: "whitehall-frontend",
+      format: "placeholder_topic",
```

Comment:
@jamiecobbett Ah, good spot. Another reason why the partial rename of `Topic` / `PolicyArea` was a Bad Thing :tm: I'll see to this.